client/my-sites/patterns/components/category-gallery/client.tsx

```diff
@@ -57,6 +57,17 @@ function CategoryGalleryItem( { category, patternTypeFilter }: CategoryGalleryIt
 			? category.pagePatternCount
 			: category.regularPatternCount;
 
+	const patternCountText =
```

Comment:
We have extra condition above, which makes the code even more complicated, check it out -- the same, but we removed a few lines with extra condition and additionally we can at a glance see that we use `regularPatternCount` for regular text and `pagePatternCount` for layouts.

And one more thing - the first condition used `=== PAGES`, and the second one used `=== REGULAR`, so we didn't have consistency around the same code.

```
const patternCountText =
patternTypeFilter === PatternTypeFilter.REGULAR
	? translate( '%(count)d pattern', '%(count)d patterns', {
			count: category.regularPatternCount,
			args: { count: category.regularPatternCount },
	  } )
	: translate( '%(count)d layout', '%(count)d layouts', {
			count: category.pagePatternCount,
			args: { count: category.pagePatternCount },
	  } );
```

Reply:
I totally see your point, thank you!